action/steps/run_tests.sh

```diff
@@ -74,19 +74,18 @@ if [ $EDIT_MODE = true ]; then
   echo "#   Testing in EditMode   #"
   echo "###########################"
   echo ""
-  xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' \
-    /opt/Unity/Editor/Unity \
-      -batchmode \
-      -logFile "$FULL_ARTIFACTS_PATH/editmode.log" \
-      -projectPath "$UNITY_PROJECT_PATH" \
-      -runTests \
-      -testPlatform editmode \
-      -testResults "$FULL_ARTIFACTS_PATH/editmode-results.xml" \
-      $CUSTOM_PARAMETERS
-      
+  unity-editor \
+    -batchmode \
+    -logFile "$FULL_ARTIFACTS_PATH/editmode.log" \
```

Comment:
`-nographics` flag needs to be added here also one more place in the same file, should that be a separate PR ?

Reply:
Up to you. I'm fine with either.